opcua/server/address_space.py

```diff
@@ -108,7 +108,8 @@ def _suitable_reftype(self, ref1, ref2, subtypes):
         """
         if ref1.Identifier == ref2.Identifier:
             return True
-        if not subtypes and ref2.Identifier == ua.ObjectIds.HasSubtype:
+        #TODO: Please check the changes here
+        elif not subtypes:
```

Comment:
@oroulet Please check this line. i change it to get all test to work. I don't understand it at all, for me the if statement before doesn't make any sense :/